samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala

```diff
@@ -56,7 +56,7 @@ class TestKafkaCheckpointManager extends KafkaServerTestHarness {
 
   val checkpointTopic = "checkpoint-topic"
   val serdeCheckpointTopic = "checkpoint-topic-invalid-serde"
-  val checkpointTopicConfig = KafkaCheckpointManagerFactory.getCheckpointTopicProperties(null)
+  val checkpointTopicConfig = new org.apache.samza.config.KafkaConfig(new MapConfig()).getCheckpointTopicProperties()
```

Comment:
need not use the fully qualified names here.

Reply:
Actually we are using the KafkaConfig from kafka-server in this class so I have to use the full name here.